src/dissectBCL/misc.py

```diff
@@ -510,7 +510,7 @@ def multiQC_yaml(flowcell, project, laneFolder):
     ))
     # Organisms
     orgs = ', '.join(list(
-        ssdf["Organism"].fillna('None').unique()
+        ssdf["Organism"].str[0].fillna('None').unique()
```

Comment:
I'm a bit puzzled by this, is it to get the "H." in the full string, e.g. "H. sapiens (GRCh37 / hg19)"? Can you share how 1 of the rows in `ssdf` look? Only if you have it around, on the tmux session... Else I can try thinking harder.. just to be sure we are on the same page, here (folowing the example) we want the string "hg19" (on parkour DB a.k.a. "label"). right?

Reply:
No, but unique() would not work with a list, i.e the result of ssdf["Organism"].fillna('None')

Reply:
["human", None, None]